packages/react-components/src/graph-visualizer/constants.ts

```diff
@@ -0,0 +1,43 @@
+export const GRAPH_COLORS = {
```

Comment:
should we import the colors from awsui design tokens so we can get light & dark mode support?

Reply:
Yes, I am switching to that. its coming in the next PR.